pkgs/development/python-modules/pysqlite3/default.nix

```diff
@@ -9,9 +9,11 @@ buildPythonPackage rec {
   pname = "pysqlite3";
   version = "0.5.2";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-CrwXVuizm9UcMD6PrDdJfMrMx7I3yJ/jz0fujmFMWQs=";
+  src = fetchFromGitHub {
+    owner = "coleifer";
+    repo = pname;
```

Comment:
I think current idea is that `repo = pname` is unwelcome. https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Fixed and thank you for the link.